source/index.html.md.erb

```diff
@@ -581,6 +585,8 @@ Here is an example of a URL to send your user as a `GET` request:
 If your user has not logged in to Smartsheet, they are first directed to the Smartsheet login page. 
 After a successful login, Smartsheet prompts your user to allow or deny the access scopes you requested.
 
+<aside class="notice">At the end of this process, your redirect URL must match what you set at the start of this flow. One trick for remembering that or if you have multiple requests is to pass the `state` value to list what redirect URL you plan to use.</aside>
```

Comment:
Please check with the dev on the email thread. I suspect the restriction is that the final url must be *under* the original. (Otherwise, what's the point?) 
Please remove the 'trick' detail here. Could be part of the 'state' description. perhaps: "State may also be used to distinguish between multiple authorization flows."

Reply:
I reworked it. Mostly using the text you suggested. But I additionally reworded your text as an aside. It's probably overkill.